packages/nodes-base/nodes/InvoiceNinja/V1u2/InvoiceNinjaTriggerV1u2.node.ts

```diff
@@ -0,0 +1,242 @@
+/* eslint-disable n8n-nodes-base/node-filename-against-convention */
+import type { IHookFunctions, IWebhookFunctions } from 'n8n-core';
+
+import type {
+	INodeType,
+	INodeTypeBaseDescription,
+	INodeTypeDescription,
+	IWebhookResponseData,
+} from 'n8n-workflow';
+
+import {
+	eventID,
+	invoiceNinjaApiRequest,
+	invoiceNinjaApiRequestAllItems,
+} from './GenericFunctions';
+
+export class InvoiceNinjaTriggerV1u2 implements INodeType {
+	description: INodeTypeDescription;
+
+	constructor(baseDescription: INodeTypeBaseDescription) {
+		this.description = {
+			...baseDescription,
+			displayName: 'Invoice Ninja Trigger',
+			name: 'invoiceNinjaTrigger',
+			icon: 'file:invoiceNinja.svg',
+			group: ['trigger'],
+			version: [1, 2],
+			description: 'Starts the workflow when Invoice Ninja events occur',
+			defaults: {
+				name: 'Invoice Ninja Trigger',
+			},
+			inputs: ['main'],
```

Comment:
@Joffcom eslint also mentioned a fix for this line: 
inputs: [] to inputs. ['main']

please check